tensorflow/python/ops/image_ops_impl.py

```diff
@@ -2805,3 +2805,26 @@ def sobel_edges(image):
   output = array_ops.reshape(output, shape=shape)
   output.set_shape(static_image_shape.concatenate([num_kernels]))
   return output
+
+
+@tf_export('image.extract_glimpse')
```

Comment:
Would it make sense to make this:

```python
@tf_export(v1=['image.extract_glimpse'], v2=[])
def ...
```

And add another function that just wraps over `gen_image_ops.extract_glimpse` to be exported for v2?